client/reader/onboarding/interests-modal/style.scss

```
@@ -4,13 +4,21 @@
 .interests-modal {
 	&__content {
 		padding: 24px;
+
+		@media ( max-width: 768px ) {
```

For these mobile breakpoints, we can use the `$break-medium` variable that's included via `@import '@wordpress/base-styles/breakpoints';`. That will break at `782px`, which is where the sidebar collapses.

```suggestion
		@media ( max-width: $break-medium ) {
```